libmachine/mcnerror/errors.go

```diff
@@ -17,15 +17,15 @@ type ErrHostDoesNotExist struct {
 }
 
 func (e ErrHostDoesNotExist) Error() string {
-	return fmt.Sprintf("Host does not exist: %q", e.Name)
+	return fmt.Sprintf("Docker machine \"%q\" does not exist. Use docker-machine ls to list machines. Use docker-machine create [OPTIONS] to add a new one.", e.Name)
```

Comment:
I'd remove the `[OPTIONS]` part &ndash; it should be the general assumption that most commands have option flags and arguments, some mandatory. If the user attempts to run `docker-machine create`, a help message will be output listing the command's usage and options.

Reply:
cool, makes sense